bootstrap/src/main/kotlin/jp/co/soramitsu/bootstrap/genesis/IrohaUtils.kt

```diff
@@ -0,0 +1,59 @@
+package jp.co.soramitsu.bootstrap.genesis
+
+import jp.co.soramitsu.crypto.ed25519.Ed25519Sha3
+import jp.co.soramitsu.iroha.java.TransactionBuilder
+import jp.co.soramitsu.bootstrap.dto.Peer
+import jp.co.soramitsu.bootstrap.error.IrohaPublicKeyError
+import java.lang.NullPointerException
+import java.security.PublicKey
+import javax.xml.bind.DatatypeConverter
+
+fun getIrohaPublicKeyFromHexString(hex: String?): PublicKey {
+    try {
+        if(hex == null) {
+            throw NullPointerException("Public key string should be not null")
+        }
+        return Ed25519Sha3.publicKeyFromBytes(DatatypeConverter.parseBase64Binary(hex))
+    } catch(e:Exception) {
+        throw jp.co.soramitsu.bootstrap.error.IrohaPublicKeyError("${e.javaClass}:${e.message}")
```

Comment:
`e` stacktrace is lost. I think it will be hard to deal with it in the future.

Reply:
logging added